### ISSUE TITLE
Investigate and fix responsiveness on student feedback submission page

### ISSUE THREAD

**Issue by ashrayjain** [TEAMMATES/repo#4595]
Although the student feedback page is responsive in some cases, a few tweaks are still required. In some other cases, (particularly, Rubric questions), the existing experience is pretty bad. I have tested a lot of variations of questions to see which ones are bad on mobile. However, if there is something I missed, please add it here. The ones I have tested are:
- Essay questions
- Numerical scale questions  (5 - 10 values)
- Rubric questions (5 - 8 cols, 4 - 5 rows)
- Team contribution questions
- Multiple choice questions (6-7 options + other, option length ~100 chars)
- Distribute questions (6-7 options, option length ~100 chars)

Basically, these are all the question types an instructor can choose from.

Additionally, long student names (up to 50 chars, a reasonable limit as suggested [here](http://stackoverflow.com/questions/8078939/what-is-the-maximum-length-of-a-facebook-name)) and long question texts (up to 300 chars) have been tested with very minor issues present.
